packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -0,0 +1,547 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:collection';
+import 'dart:math';
+import 'package:flutter/foundation.dart';
```

Comment:
trivial nit: put a newline between the dart: and package: imports
trivial nit: import math `as math` for consistency with other parts of the framework